contrib/devtools/test-security-check.py

```diff
@@ -47,24 +50,26 @@ def test_ELF(self):
     def test_PE(self):
         source = 'test1.c'
         executable = 'test1.exe'
-        cc = 'x86_64-w64-mingw32-gcc'
+        cc = determine_wellknown_cmd('CC', 'x86_64-w64-mingw32-gcc')
         write_testcode(source)
 
-        self.assertEqual(call_security_check(cc, source, executable, ['-Wl,--no-nxcompat','-Wl,--no-dynamicbase','-Wl,--no-high-entropy-va','-no-pie','-fno-PIE']),
+        self.assertEqual(call_security_check(cc, source, executable, ['-Wl,--no-nxcompat','-Wl,--disable-reloc-section','-Wl,--no-dynamicbase','-Wl,--no-high-entropy-va','-no-pie','-fno-PIE']),
```

Comment:
At this stage we have already given in to not being able to run the test security check target for windows in gitian due to lack of `--no` options in ld, so adding `--disable` here to test `--enable-reloc-section` isn't making anything worse. If anything this speaks to the usefulness of Guix, given how easy it is to patch these `--no/--disable` flags back into our toolchain. It would be much  more difficult trying to achieve the same using gitian.